### PR TITLE
FOLIO-491: Playbooks to support curriculum

### PR DESCRIPTION
**Pull request by wafschneider**
Also review and edits to curriculum text

### CODE REVIEW

File: 04_set_up_the_folio_users_app.md [hunk 1]
```diff
@@ -148,7 +149,7 @@ $ mvn install
 ```
 
 ### Register and Deploy the Users app Okapi Module
-The Git repository for Okapi Module has a [Module Descriptor](https://github.com/folio-org/mod-users/blob/master/ModuleDescriptor.json) and a [Deployment Descriptor](https://github.com/folio-org/mod-users/blob/master/DeploymentDescriptor.json) that can be used to register and deploy the Users app Okapi Module.
+The Git repository for the Users app Okapi Module has a [Module Descriptor](https://github.com/folio-org/mod-users/blob/master/ModuleDescriptor.json) that can be used to register the Users app Okapi Module.
```
Comment (dltj) [inline 04_set_up_the_folio_users_app.md:152]: I saw that there was a [pull request on folio-org/mod-users](https://github.com/folio-org/mod-users/pull/13) that removed the excess key-value pairs from the Deployment Descriptor.  Even with those the Deployment Descriptor worked.  Is there any reason not to use the one from mod-users (with our without the extra lines)?

File: _config.yml [hunk 2]
```diff
@@ -6,3 +6,4 @@ exclude:
   - Gemfile
   - Gemfile.lock
   - vendor
+  - playbooks
```
Comment (dltj) [inline _config.yml:9]: Just commenting to say how much I :heart: Ansible.

**Comment by dltj** [comment]
Thanks @wafschneider!

File: 04_set_up_the_folio_users_app.md (hunk 1 above)
Comment (wafschneider) [reply 04_set_up_the_folio_users_app.md:152]: Just to document - the extra pairs cause Okapi to attempt to deploy on port 9131. If you follow lesson 3 before you do lesson 4, you end up with okapi-test-auth-module running on 9131, so the deployment of mod-users fails.

**Pull request merged**